packages/rum/src/resourceUtils.ts

```diff
@@ -67,15 +59,37 @@ export function computePerformanceResourceDetails(
     return undefined
   }
   return {
-    connect: formatTiming(entry.connectStart, entry.connectEnd),
-    dns: formatTiming(entry.domainLookupStart, entry.domainLookupEnd),
+    connect: isRelevantTiming(entry.connectStart, entry.connectEnd, entry.fetchStart)
+      ? formatTiming(entry.connectStart, entry.connectEnd)
+      : undefined,
+    dns: isRelevantTiming(entry.domainLookupStart, entry.domainLookupEnd, entry.fetchStart)
+      ? formatTiming(entry.domainLookupStart, entry.domainLookupEnd)
+      : undefined,
     download: formatTiming(entry.responseStart, entry.responseEnd),
     firstByte: formatTiming(entry.requestStart, entry.responseStart),
-    redirect: entry.redirectStart !== 0 ? formatTiming(entry.redirectStart, entry.redirectEnd) : undefined,
+    redirect: isRelevantTiming(entry.redirectStart, entry.redirectEnd, 0)
+      ? formatTiming(entry.redirectStart, entry.redirectEnd)
+      : undefined,
     ssl: entry.secureConnectionStart !== 0 ? formatTiming(entry.secureConnectionStart, entry.connectEnd) : undefined,
```

Comment:
Why not using `isRelevantTiming` here?

Reply:
`secureConnectionStart` and `connectEnd` do not have the same relationship as the other timings.
In most of the cases 
```js
isRelevantTiming(entry. secureConnectionStart, entry.connectEnd, 0)
```
 should be true since `connectEnd` should be > 0

Reply:
A cached resource coming from a SSL connection looks like this:
![Screenshot 2020-03-05 at 18 15 30](https://user-images.githubusercontent.com/61787/76006800-5005ea80-5f0d-11ea-8110-840af709d5b5.png)
I think that if we follow the same logic, we'd want
```ts
entry.secureConnectionStart !== 0 &&
   isRelevantTiming(entry.secureConnectionStart, entry.connectEnd, entry.fetchStart)
```

Also, I think we should use `isRelevantTiming` for `download` and `firstByte` too.

Reply:
> A cached resource coming from a SSL connection 
Yes, I missed that case

> Also, I think we should use isRelevantTiming for download and firstByte too
What cases do you see here?